boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -170,11 +170,11 @@
 		#size-cells = <1>;
 
 		cpuapp_slot0_partition: partition@a6000 {
-			reg = <0xa6000 DT_SIZE_K(512)>;
+			reg = <0xa6000 DT_SIZE_K(296)>;
```

Comment:
You should also update accordingly:
https://github.com/zephyrproject-rtos/zephyr/blob/8d344cc9d8484767f4d72a0835b5e12a614b5e58/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.yaml#L13

Reply:
Aligned!